src/commands/violations/createadvanced.js

```diff
@@ -41,11 +41,12 @@ class CreateReportAdvanced extends Command {
 		let proof = (await getMessageResponse(message, "Please send a link to proof of the report or `none` if there is no proof"))?.content
 		if (proof.toLowerCase() === "none") proof = undefined
 
-		let timestamp = (await getMessageResponse(message, "Please send a value representing the date of the report. Type in `now` to set the current time"))?.content
-		if (timestamp.toLowerCase() === "now") timestamp = (new Date).toISOString()
+		const timestampMsg = (await getMessageResponse(message, "Please send a value representing the date of the report. Type in `now` to set the current time"))?.content
+		let timestamp = new Date()
+		if (timestampMsg.toLowerCase() === "now") timestamp = new Date()
 		else {
-			if (isNaN(Date.parse(timestamp))) timestamp = (new Date).toISOString()
-			else timestamp = Date.parse(timestamp).toISOString()
+			if (isNaN(Date.parse(timestampMsg))) timestamp = new Date()
+			else timestamp = new Date(timestampMsg)
```

Comment:
By discarding the result of Date.parse you are making it preform it a second time during Date().
Would the better method not be to directly assign the result of parse to timestamp, and if it is NaN then set it to the current time after this initial assignment.
Even still it might be best to throw an error to the user, as if it was bad formatting and not "now" then using the current time would not be expected.

Reply:
I send an info message rather than throwing an error. Might change that later though